nautobot_firewall_models/tests/test_capirca.py

```diff
@@ -1,16 +1,18 @@
 """Test Capirca Utils."""
-# flake8: noqa: F403,405
+
+# flake8: noqa: F403, F405
```

Comment:
```suggestion
# noqa: F403, F405
```

Reply:
I am pretty sure I tried this, and it didn't work, but I will try again.